src/officehours_api/consumers.py

```diff
@@ -140,9 +142,17 @@ def trigger_queue_update_for_meeting(sender, instance: Meeting, **kwargs):
 
 
 @receiver(m2m_changed, sender=Queue.hosts.through)
-def trigger_queue_update_for_hosts(sender, instance, action, **kwargs):
-    if action == "post_remove" or action == "post_clear" or action == "post_add":
+def trigger_queue_update_for_hosts(sender, instance, action, pk_set, **kwargs):
+    if action not in ["post_remove", "post_clear", "post_add"]:
+        return
+    if isinstance(instance, Queue):
         transaction.on_commit(lambda: send_queue_update(instance.id))
+        for host_id in pk_set:
+            transaction.on_commit(lambda: send_user_update(host_id))
+    else:  # is User
+        transaction.on_commit(lambda: send_user_update(instance.id))
+        for queue_id in pk_set:
+            transaction.on_commit(lambda: send_queue_update(queue_id))
```

Comment:
I wasn't able to trigger this, but I was trying to follow the example from `trigger_user_update_for_meetings`. Do you think this is necessary?

Reply:
I was able to trigger it by making the m2m change from and saving the user (instead of the queue), from the console. It's not strictly necessary since that's a code path that isn't used anywhere in the app right now, but since it could come in handy later, I'd say let's leave it.